src/Plugin/AssetMinificationPlugin.php

```diff
@@ -14,6 +14,6 @@ class AssetMinificationPlugin
 
     public function afterGetExcludes($subject, array $result, string $contentType): array
     {
-        return array_merge($result, $contentType === 'js' ? [self::JS_LIBRARY] : []);
+        return [...$result, ...$contentType === 'js' ? [self::JS_LIBRARY] : []];
```

Comment:
Since $contentType is either array with only one element or even empty array i think it does not have much sense to use spread operator here.

I think easier to read will be 
`return $result + $contentType === 'js' ? [self::JS_LIBRARY] : [];`

Reply:
I have test the solution which you recommend but it throws an exception as below:
![image](https://user-images.githubusercontent.com/11246832/151394564-e3b2c24b-9952-4c5d-b5ec-09032fd131bb.png)

Reply:
Put the second operand in parentheses

Reply:
done